ArduPlane/system.cpp

```diff
@@ -336,6 +336,9 @@ void Plane::set_mode(enum FlightMode mode)
     crash_state.is_crashed = false;
     crash_state.impact_detected = false;
 
+    // reset external attitude guidance
+    guided_state.guiding_roll = false;
```

Comment:
after thinking about this, we could easily do this with a single variable, the boolean is not needed.

set the timer to 0 here to clear/reset, and when using it just check for non-zero